eng/dotnet.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
+)
+
+. $PSScriptRoot\common\tools.ps1
+
+try {
+  # Import custom tools configuration, if present in the repo.
+  # Note: Import in global scope so that the script set top-level variables without qualification.
+  $configureToolsetScript = Join-Path $EngRoot "configure-toolset.ps1"
+  if (Test-Path $configureToolsetScript) {
+    . $configureToolsetScript
+  }
+
+  # Initialize dotnet cli if listed in 'tools'
+  $dotnetRoot = $null
+  if (Get-Member -InputObject $GlobalJson.tools -Name "dotnet") {
+    $dotnetRoot = InitializeDotNetCli -install:$restore
+    $dotnet = Join-Path $dotnetRoot "dotnet.exe"
+  }
+
+  if (!$dotnet) {
+    Write-Host "/global.json must specify 'tools.dotnet'." -ForegroundColor Red
+    ExitWithExitCode 1
+  }
+
+  $buildTool = @{ Path = Join-Path $dotnetRoot "dotnet.exe"; Command = "msbuild" }
```

Comment:
What is this doing? I don't see buildTool used anywhere.

Reply:
It looks like this is doing some of what [InitializeBuildTool](https://github.com/dotnet/arcade/blob/1275e8827a5d7963811540f43f71ab35a1a6412a/eng/common/tools.ps1#L322) does. Could we just call that function instead? In either case, I think the logic here and in dotnet.sh should be the same.

Reply:
I was thinking that call did too much, but ….  I just had to add the call to InitializeBuildTool on the bash side.  So I think that would be best to match.